src/configs/index.js

```diff
@@ -287,10 +313,19 @@ export const config = {
       available: 'Доступний',
       handmade: 'Зроблений вручну',
       avatarText: 'Фото'
+    },
+    model: {
+      image: 'Фото гобелена',
+      category: 'Категорія',
+      available: 'Доступний',
+      show: 'Показати',
+      avatarText: 'Фото',
+      priority: 'Пріорітет'
     }
   },
   IMG_URL: 'https://horondi.blob.core.windows.net/horondi/images/',
   patternImageLink: `https://horondi.blob.core.windows.net/horondi/images/`,
+  imageLink: `https://horondi.blob.core.windows.net/horondi/images/`,
```

Comment:
This value is already assigned on line `326`, do we really need to duplicate it?